VideoLocker/res/values/styles.xml

```diff
@@ -23,9 +23,11 @@
         <item name="drawerArrowStyle">@style/DrawerArrowStyle</item>
         <item name="actionBarTabStyle">@style/TabsStyle</item>
         <item name="disableableButtonTextColor">@color/disableable_button_text</item>
+        <item name="android:listChoiceBackgroundIndicator">@drawable/list_item_selector</item>
+        <item name="listChoiceBackgroundIndicator">@drawable/list_item_selector</item>
```

Comment:
shouldn't this declaration be enough ?
why do we need `android:listChoiceBackgroundIndicator` ?